cdap-client-tests/src/test/java/co/cask/cdap/client/NamespaceClientTestRun.java

```diff
@@ -121,21 +121,9 @@ public void testDeleteAll() throws Exception {
     Assert.assertEquals(Id.Namespace.DEFAULT.getId(), namespaceClient.list().get(0).getName());
   }
 
-  private void verifyDoesNotExist(Id.Namespace namespaceId)
-    throws Exception {
-
-    try {
-      namespaceClient.get(namespaceId);
-      Assert.fail(String.format("Namespace '%s' must not be found", namespaceId));
-    } catch (NotFoundException e) {
-      // expected
-    }
-
-    try {
-      namespaceClient.delete(namespaceId);
+  private void verifyDoesNotExist(Id.Namespace namespaceId) throws Exception {
+    if (namespaceClient.exists(namespaceId)) {
```

Comment:
can just do ``Assert.assertFalse(String.format("Namespace '%s' must not be found", namespaceId), namespaceClient.exists(namespaceId))``